bundles/org.openhab.binding.hapero/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,546 @@
+				<option value="0">Standby</option>
+				<option value="1">Air check</option>
+				<option value="2">Combustion start</option>
+				<option value="3">Ember buildup</option>
+				<option value="4">Controlled operation</option>
+				<option value="5">Combustion stop</option>
+				<option value="6">Ignition detection</option>
+				<option value="7">Maximum Air</option>
+				<option value="8">Cooling</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="furnaceGrateStatus" advanced="true">
+		<item-type>String</item-type>
+		<label>Burner Grate Status</label>
+		<description>Operating status of the burner grate drive</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">Standby</option>
+				<option value="1">Initializing</option>
+				<option value="2">Starting</option>
+				<option value="3">Ignition detection</option>
+				<option value="4">Grate cleaning</option>
+				<option value="5">Heating</option>
+				<option value="6">Opening</option>
+				<option value="7">Full open/close</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="furnaceErrorStatus">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Error Code</label>
+		<description>Last error code of the furnace controller</description>
+	</channel-type>
+	<channel-type id="furnaceMultifunctionMotorMode" advanced="true">
+		<item-type>String</item-type>
+		<label>Multifunction Motor Mode</label>
+		<description>Operating mode of the multi function motor</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">Idle</option>
+				<option value="1">Motor Stop</option>
+				<option value="2">Cleaning</option>
+				<option value="3">Heating position</option>
+				<option value="4">Standby position</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="furnaceMultifunctionMotorStatus" advanced="true">
+		<item-type>String</item-type>
+		<label>Multifunction Motor Status</label>
+		<description>Operating status of the multi function motor</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">Position Unknown</option>
+				<option value="1">Heating position</option>
+				<option value="2">Standby position</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="furnacePowerChannel">
+		<item-type>Number</item-type>
+		<label>Power Level</label>
+		<description>Power level of the furnace</description>
+		<state min="0" max="100" step="0.1" pattern="%.1f %%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="furnaceAirFlow" advanced="true">
+		<item-type>Number</item-type>
+		<label>Air Flow</label>
+		<description>Combustion air flow</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="furnaceAirFlowSetpoint" advanced="true">
+		<item-type>Number</item-type>
+		<label>Target Air Flow</label>
+		<description>Target Air flow</description>
+		<tags>
+			<tag>Setpoint</tag>
+			<tag>Temperature</tag>
+		</tags>
```

Comment:
This is not temperature, but air flow? I didn't check if such tag exist, but otherwise please remove.

Reply:
Tags removed.